### ISSUE TITLE
[bug] after resize the image is even more big 

### ISSUE THREAD

**Issue by xiaochong0302** [phalcon/cphalcon#10908]
```php
$image = new \Phalcon\Image\Adapter\Imagick("test.jpg");
$image->resize(800, 600);
$image->save();
```

picture 1:

before resize: 66K (1024 * 520)
after resize: 75K (800 * 406)

picture 2:

before resize: 218K (1920 * 1285)
after resize: 338K (800 * 535)

Do i miss something ?

[https://forum.phalconphp.com/discussion/8299/after-resize-the-image-is-even-more-big](https://forum.phalconphp.com/discussion/8299/after-resize-the-image-is-even-more-big)

**Comment by andresgutierrez** [comment]
@dreamsxin Could you please check this?

**Comment by dreamsxin** [comment]
@andresgutierrez  OK

**Comment by dreamsxin** [comment]
@xiaochong0302 你可以测试下我这个分支么？

**Comment by andresgutierrez** [comment]
Fixed in the 2.1.x branch
